README.md

```diff
@@ -90,6 +90,11 @@ Name                                               | Leads
 [rfc-2229-stream]: https://rust-lang.zulipchat.com/#narrow/stream/189812-t-compiler.2Fwg-rfc-2229
 [mir-opt-stream]: https://rust-lang.zulipchat.com/#narrow/stream/189540-t-compiler.2Fwg-mir-opt
 
+## Expert Map
+
+The [experts directory](experts) contains a map of areas of the compiler
+to experts from the Rust compiler team.
```

Comment:
The "experts directory" link seems to be broken (old pull request?).

Reply:
I think it is just copied from the PR diff where it would correctly link to the `experts` directory in this repository when rendered in a markdown document.